app/uk/gov/nationalarchives/omega/editorial/models/package.scala

```diff
@@ -57,6 +57,20 @@ package object editSetRecords {
         |      "description" : "Bedlington Colliery, Newcastle Upon Tyne. Photograph depicting: view of pithead baths."
         |    }
         |  ],
+        |  "separatedMaterial" : [
```

Comment:
I recommend that we have a variety of item types (e.g. some with a link, some with a description) whether in one record or (preferably) across several records; if only to better demonstrate that variety.

Reply:
Confirmed.

Reply:
I've added different separated material types to records 10, 11 and 12